README.md

```diff
@@ -4,7 +4,7 @@ This repository contains dotnet samples for integration with the Lenus Health pl
 
 ## Requirements
 
-- .NET core 3.1 SDK
+- .NET core 7.0 SDK
 - Visual Studio 2019 or Visual Studio Code
```

Comment:
Can you bump it to 2022 so we don't sound like dinosaurs? 😂

Reply:
Done.